rke/schema_rke_cluster_system_images.go

```diff
@@ -132,6 +132,30 @@ func rkeClusterSystemImagesFields() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Optional: true,
 		},
+		"aci_cni_deploy_container": {
+			Type:     schema.TypeString,
+			Optional: true,
+		},
+		"aci_host_container": {
+			Type:     schema.TypeString,
+			Optional: true,
+		},
+		"aci_opflex_container": {
+			Type:     schema.TypeString,
+			Optional: true,
+		},
+		"aci_mcast_container": {
+			Type:     schema.TypeString,
+			Optional: true,
+		},
+		"aci_ovs_container": {
+			Type:     schema.TypeString,
+			Optional: true,
+		},
+		"aci_controller_container": {
+			Type:     schema.TypeString,
+			Optional: true,
+		},
```

Comment:
Checking [rke system images](https://github.com/rancher/rke/blob/release/v1.2/types/rke_types.go#L189), are `AciGbpServerContainer` and `AciOpflexServerContainer ` not required as configurable images??

Reply:
These images are to support the overlay case. On rke upstream, the validator doesn't check for these images yet. After we push the changes to support overlay case for ACI CNI on rke side, I will incorporate the changes to include AciGbpServerContainer and AciOpflexServerContainer on terraform.